### ISSUE TITLE
JS files not linted although included by project configuration

### ISSUE THREAD

**Issue by ilkka** [palantir/tslint#2176]
### Bug Report

- __TSLint version__: `4.4.2`
- __TypeScript version__: `2.1.5`
- __Running TSLint via__: CLI

#### TypeScript code being linted

Example project: https://github.com/ilkka/tslint-js-test

#### Actual behavior

`tslint --project .` does not flag incorrect quotes in `src/index.js` although `tslint src/index.js` does.

#### Expected behavior

`tslint --project .` flags the single quotes as incorrect, as configured.

It does not seem to make a difference whether the project source configuration is done via `files` or `include`.

**Comment by JoshuaKGoldberg** [comment]
I'm no longer able to reproduce this as of TypeScript 3.1.3 with the following setup:

`tsconfig.json`:

```json
{
  "compilerOptions": {
    "allowJs": true
  },
  "include": ["*.js", "*.ts"]
}

`tslint.json`:

```json
{
    "jsRules": {
        "no-var-keyword": true
    }
}
```

`index.js`:

```javascript
var wat = 7;
```

```powershell
PS C:\Code\test> tslint -c tslint.json -p tsconfig.json
No valid rules have been specified for TypeScript files

ERROR: C:/Code/test/untyped.js[1, 1]: Forbidden 'var' keyword, use 'let' or 'const' instead
```

Looks like it's been fixed! 🎉
